shell/platform/windows/flutter_windows_view.cc

```diff
@@ -352,6 +352,7 @@ void FlutterWindowsView::SendWindowMetrics(size_t width,
   event.width = width;
   event.height = height;
   event.pixel_ratio = dpiScale;
+  event.view_id = view_id();
```

Comment:
These can just use the private field directly:

```suggestion
  event.view_id = view_id_;
```